lib/routemaster/api_client.rb

```diff
@@ -33,7 +33,18 @@ def post(url, body: {}, headers: {})
         connection.post do |req|
           req.url url
           req.headers = headers.merge(auth_header(host))
-          req.body = body.to_json
+          req.body = body
+        end
+      end
+    end
+
+    def patch(url, body: {}, headers: {})
+      host = URI.parse(url).host
+      response_wrapper do
+        connection.patch do |req|
+          req.url url
+          req.headers = headers.merge(auth_header(host))
```

Comment:
This gets repeated everywhere; we'll probably want to extract it to a middleware eventually.